### PR TITLE
Move meta tests into infrastructure/tooling

### PR DESCRIPTION
**Pull request by djmitche**
..and use console-taskgraph for them.  This isn't much more than cleanup.

### CODE REVIEW

**Comment by owlishDeveloper** [comment]
can haz no red crossez?

**Comment by djmitche** [comment]
That's a lot of them!  But I'll look.

File: infrastructure/tooling/src/meta/checks/depcheck.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+const util = require('util');
+const depcheck = require('depcheck');
+const exec = util.promisify(require('child_process').exec);
+const _ = require('lodash');
+const {REPO_ROOT} = require('../../utils');
+
+/*
+ * The 'depcheck' tool is async but still blocks for long stretches, perhaps
+ * doing computation.  So, we defer that to a worker thread.
+ */
```
Comment (imbstack) [inline infrastructure/tooling/src/meta/checks/depcheck.js]: I love that our general view on web endpoint timings is `¯\_(ツ)_/¯` but when it comes to our tooling we are going out of our way to make it performant.

File: infrastructure/tooling/src/meta/checks/speling.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const util = require('util');
```
Comment (imbstack) [inline infrastructure/tooling/src/meta/checks/speling.js]: hahaha, `speling.js`

File: infrastructure/tooling/src/meta/checks/depcheck.js (hunk 1 above)
Comment (djmitche) [reply infrastructure/tooling/src/meta/checks/depcheck.js]: It's worse than that! I was annoyed that the console-taskgraph spinny things weren't spinning!

Also, this was a chance to play with worker threads in a non-scary context.

**Pull request merged**